corgi-concourse-resource/corgi_concourse_resource/utils.py

```diff
@@ -2,6 +2,20 @@
 import sys
 
 
+def get_collection_id(job):
+    repo = job["repository"]
+    book_slug = job["books"][0]["slug"]
+    owner = repo["owner"]
+    repo_name = repo["name"]
+    if owner != "openstax":  # pragma: no cover
+        return f"{owner}/{repo_name}/{book_slug}"
+    return f"{repo_name}/{book_slug}"
```

Comment:
```suggestion
    return f"{owner}/{repo_name}/{book_slug}"
```

Why special-case openstax? This way the "collection_id" must always have _exactly_ 2 slashes in it :joy:

Reply:
I will give that a try. Some of this was just me mindlessly copying and making necessary edits.